src/styles/bloks/_card.scss

```
@@ -0,0 +1,106 @@
+.catalog {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: center;
+  box-sizing: content-box;
+  max-width: ($card-with + 48px) * 4;
+
+  &__card {
+    width: $card-with;
+    margin: 0 48px 46px 0;
+    padding: 32px 16px 16px;
+  }
+}
+
+.card {
+  color: $main-color;
+  background-color: $card-background-color;
+  border: 1px solid #f3f3f3;
```

You created variables for colors, but not for all 🥺 
```suggestion
  border: 1px solid $c-color;
```